src/platform/plugins/shared/discover/public/application/main/state_management/redux/actions/tabs.ts

```diff
@@ -86,17 +95,22 @@ export const updateTabs: InternalStateThunkActionCreator<[TabbedContentState], P
       const tab: TabState = {
         ...defaultTabState,
         ...existingTab,
-        ...pick(item, 'id', 'label'),
+        ...pick(item, 'id', 'label', 'duplicatedFromId'),
       };
 
-      if (item.duplicatedFromId) {
-        const existingTabToDuplicate = selectTab(currentState, item.duplicatedFromId);
+      const existingTabToDuplicateFrom =
+        item.duplicatedFromId && !existingTab
+          ? selectTab(currentState, item.duplicatedFromId)
+          : undefined;
+      if (item.duplicatedFromId && existingTabToDuplicateFrom) {
```

Comment:
```suggestion
      if (existingTabToDuplicateFrom) {
```
Nit: not sure this extra condition actually does anything anymore.